keyboards/gherkin/keymaps/wanleg/keymap.c

```diff
@@ -14,6 +14,48 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "gherkin.h"
+////////// ProMicro Bootloader Section START //////////
+//this section can be used if the "RESET" keycode doesn't work for some reason
+//shouldn't hurt anything if you don't use it, and you can delete it if you don't want it
+#include <avr/wdt.h>
+/* id for user defined functions */
+enum function_id {
+    PROMICRO_PROGRAM,
+};
+
+void promicro_bootloader_jmp(bool program) {
```

Comment:
This code is redundant.

As long as the board has `BOOTLOADER = caterina` defined in it's rules.mk, it should be doing this with the normal reset command. 

https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c#L135-L152